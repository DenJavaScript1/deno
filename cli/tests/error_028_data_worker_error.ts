// throw Error("bad");
const worker = new Worker('data:application/javascript;base64,dGhyb3cgRXJyb3IoImJhZCIpOw==', { type: "module" });
