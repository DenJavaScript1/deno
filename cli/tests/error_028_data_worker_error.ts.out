[WILDCARD]error: Uncaught Error: bad
  throw Error("bad");
        ^
    at data:application/javascript;base64,dGh...pOw==:2:7)